### ISSUE TITLE
Canvas option only displays first canvas

### ISSUE THREAD

**Issue by apires03** [jasonday/printThis#169]
Description of issue:
It seems the canvas option is not working if you have multiple canvas that prints separately.  Issue is that the code is not removing the markup. The issue is on line 234:
```
// Remove the markup from the original
$src.removeData('printthis');

```
The fix for it is:
```
// Remove the markup from the original
if ($.isFunction($.removeAttr)) {
	$($src[0]).removeAttr('data-printthis');
} else {
	$src[0].removeAttribute('data-printthis');
}
```
====================================================================

Code & options:
```
//Example:
$('#myDiv').printThis({
   canvas: true
});
 ```

**Comment by jasonday** [comment]
Sorry, we haven't looked at this. 

@oculus42 I think we're butting up against this:

> When using .removeData("name"), jQuery will attempt to locate a data- attribute on the element if no property by that name is in the internal data cache. To avoid a re-query of the data- attribute, set the name to a value of either null or undefined (e.g. .data("name", undefined)) rather than using .removeData().

Related: https://stackoverflow.com/questions/16615383/removedata-jquery-method-not-working/19205450

**Comment by jasonday** [comment]
More specific from the docs:

>  Removing data from jQuery's internal .data() cache does not effect any HTML5 data- attributes in a document; use .removeAttr() to remove those.

**Comment by oculus42** [comment]
Validated on jsfiddle: https://jsfiddle.net/xpvt214o/921932/
1.13.0 coming shortly.

**Comment by oculus42** [comment]
Published 1.14.1 to npm.
